docs/website/docs/dlt-ecosystem/destinations/databricks.md

```diff
@@ -15,7 +15,85 @@ keywords: [Databricks, destination, data warehouse]
 pip install dlt[databricks]
 ```
 
-## Setup Guide
+## Set up your Databricks workspace
+
+To use the Databricks destination, you need:
+
+* A Databricks workspace with a Unity Catalog metastore connected
+* A Gen 2 Azure storage account and container
+
+If you already have your Databricks workspace set up, you can skip to the [Loader setup Guide](#loader-setup-guide).
+
+### 1. Create a Databricks workspace in Azure
+
+1. Create a Databricks workspace in Azure
+
+    In your Azure Portal search for Databricks and create a new workspace. In the "Pricing Tier" section, select "Premium" to be able to use the Unity Catalog.
+
+2. Create a storage account
+
+    Search for "Storage accounts" in the Azure Portal and create a new storage account. Make sure to select "StorageV2 (general purpose v2)" as the account kind.
+
+3. Create a container in the storage account
+
+    In the storage account, create a new container. This will be used as a datastore for your Databricks catalog.
+
+4. Create an Access Connector for Azure Databricks
+
+    This will allow Databricks to access your storage account.
+    In the Azure Portal search for "Access Connector for Azure Databricks" and create a new connector.
```

Comment:
You could add a note here that users can also use the Access Connector for Azure Databricks that gets created by default in the Databricks managed resource group when creating a new Databricks workspace.

Reply:
Where do I find this? Guess I'm lacking permissions :thinking:

Reply:
It's in the managed resource group that gets created automatically when you create a Databricks workspace, but we don't have permissions to see it. We had a little chat about it yesterday:

![image](https://github.com/dlt-hub/dlt/assets/47451109/616baff1-7fdf-472f-8596-995a4d67d1fb)